src/sentry/api/serializers/rest_framework/mentions.py

```diff
@@ -0,0 +1,31 @@
+from __future__ import absolute_import
+
+from sentry.api.fields.actor import Actor
+from sentry.api.serializers.rest_framework.group_notes import seperate_resolved_actors
+from sentry.models import User
+
+
+def extract_user_ids_from_mentions(organization_id, mentions):
```

Comment:
Why is this part of the serializers modules? Seems like this code is general application logic. :bike:

Reply:
I kind of agree, but at the moment the `Actor` model lives in the api, and is basically used to translate api input into an object, so that part basically belongs in the api.

I could see an argument that fetching all users for a given set of teams is general app logic though. Not sure where it should go though?

Reply:
Under teams? You're converting teams into the list of members they contain.